core/src/components/card/card.scss

```diff
@@ -70,6 +70,8 @@
   @include padding(0);
   @include margin(0);
 
+  display: block;
```

Comment:
The issue here stems from `img` being displayed `inline` by default. Can you apply `display: block` to the `img` element instead of `:host` here?

Reply:
Hm, I'm not sure to be honest. I may be misreading your comment but the `<img>` tag is already displayed as a block level element (https://github.com/ionic-team/ionic/blob/master/core/src/components/card/card.scss#L44-L48).

The issue currently is we have content like this
```html
<ion-card>
  <button>
    <img src=".." />
  </button>
</ion-card>
```

The `button` element being inline block will render the space between it's closing tag and the closing ion-card type which results in a gap between the bottom of the button and the bottom of the card.

`<ion-card>`
<img width="314" alt="Screenshot 2019-05-22 at 15 15 00" src="https://user-images.githubusercontent.com/846587/58183806-da939b00-7ca7-11e9-8c66-b0b4972cdf83.png">

`<button>`
<img width="314" alt="Screenshot 2019-05-22 at 15 14 54" src="https://user-images.githubusercontent.com/846587/58183805-da939b00-7ca7-11e9-8625-18996cdb666c.png">

If we make just the wrapping button a block level element it'll ignore the white space between closing tags and always match the height of the card which is what we want I think?

Reply:
Huh interesting... We had a similar issue on `ion-img`, and the solution there was to apply `display` to the `img` element. I took a look at the code and what you have is probably fine. 👍